
The Toshiba Equium A6 Laptop needs pci-assign-busses in order to detect
the Wireless card Netgear MA521

More info here: http://qa.mandriva.com/show_bug.cgi?id=18989

Signed-off-by: Thomas Backlund <tmb@mandriva.org>

---
 arch/x86/pci/common.c |    8 ++++++++
 1 file changed, 8 insertions(+)

--- a/arch/x86/pci/common.c
+++ b/arch/x86/pci/common.c
@@ -196,6 +196,14 @@ static struct dmi_system_id __devinitdat
 			DMI_MATCH(DMI_PRODUCT_NAME, "SX20S"),
 		},
 	},
+	{
+		.callback = assign_all_busses,
+		.ident = "Toshiba Equium A6 Laptop",
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "TOSHIBA"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "Equium A60"),
+		},
+	},
 #endif		/* __i386__ */
 	{
 		.callback = set_bf_sort,
